template.tpl

```diff
@@ -597,6 +611,23 @@ scenarios:
       onSuccess();
     });
 
+    runCode(mockData);
+    assertApi('gtmOnSuccess').wasCalled();
+- name: No API - Test sendPixel with eventId
+  code: |-
+    const encodeUriComponent = require('encodeUriComponent');
+    const getUrl = require('getUrl');
+    const mockData = {
+      partnerId: '123',
+      conversionId: '12576358',
+      eventId: 'uniqueEventId123'
+    };
+
+    mock('sendPixel', (url, onSuccess, onFailure) => {
```

Comment:
Thanks for the test. 
I have been thinking, since there is no way of automatic validation of this GitHub repo with the GTM testing console, we should just add screenshots of passing tests. WDYT? I plan to do that from now on too.

Reply:
I agree, that is a good point. I did add a screenshot of all tests passing in the PR, here is another screenshot with a run from just this test:

<img width="1764" alt="Screen Shot 2023-04-11 at 10 30 36 AM" src="https://user-images.githubusercontent.com/43223664/231242678-0eba5834-a05b-4061-bc4b-9be1d33ea29b.png">

> since there is no way of automatic validation of this GitHub repo with the GTM testing console

Btw, do you know if is there no way to achieve this or do we just not have it for this repo?

Reply:
We can have some test integration build scripts, but outside of GTM console, we can’t get their env. for tests.